pkg/api/volopt_level.go

```diff
@@ -0,0 +1,29 @@
+package api
+
+// OptionLevel is the level at which option is visible to users
+//go:generate jsonenums -type=OptionLevel
+type OptionLevel uint
+
+// These are the available option levels
+const (
+	OptionStatusAdvanced OptionLevel = iota
+	OptionStatusBasic
+	OptionStatusExperimental
+	OptionStatusDeprecated
+	OptionStatusUndefined
+)
+
+func (l OptionLevel) String() string {
+	switch l {
+	case OptionStatusBasic:
+		return "Basic"
```

Comment:
What's returned in API response can be small case.

Reply:
This is an auto-generated file, not a good idea to modify